_maps/_basemap.dm

```diff
@@ -1,15 +1,8 @@
 //#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
 
 #include "map_files\generic\CentCom.dmm"
-#include "map_files\generic\SpaceStation.dmm"
 
 #ifndef LOWMEMORYMODE
-
-	#include "map_files\generic\Space.dmm"
-	#include "map_files\generic\Space2.dmm"
-	#include "map_files\Mining\Lavaland.dmm"
-	#include "map_files\generic\City_of_Cogs.dmm"
```

Comment:
Move these down to all maps

Reply:
Added Lavaland and City_of_Cogs to the ALL_MAPS section.